stock_cycle_count/models/stock_location.py

```diff
@@ -115,9 +115,9 @@ def action_accuracy_stats(self):
         self.ensure_one()
         action = self.env.ref("stock_cycle_count.act_accuracy_stats")
         result = action.read()[0]
-        result["context"] = {"search_default_location_id": self.id}
+        result["context"] = {"search_default_location_ids": self.name}
```

Comment:
I doubt this is correct, as you are mixing text with IDs. In any case, it should be `self.ids`.

Reply:
The reason: this text is pass to search view as input, `ids` not working.

Reply:
But that's wrong. Having other locations with similar name, they will be filtered in here.

Reply:
@pedrobaeza  If pass `ids ` is wrong on `searchview`
![image](https://user-images.githubusercontent.com/7775116/133947618-752e8c67-7f5c-456f-8bfa-49743233dcb6.png)

I update code for pass complete_name, is OK now??

Reply:
After change
![image](https://user-images.githubusercontent.com/7775116/133947877-f42c278e-a939-4d30-ae1f-af9df4e1a153.png)

Reply:
The problem is that you can't search by IDs on a m2m, but I think you should convert the domain to `id in ...` instead.

Reply:
I Update my code, is OK now?